### PR TITLE
Trafficinfo 510.generate.dashboards

### PR DESCRIPTION
**Pull request by dniel**
Add the grafana module, to generate Grafana dashboard per service, if enabled.
* by default disabled

Depends on the Grafana provider to create grafana resources.
The provider needs to be added to the top-level module in the service that use ecs-microservice module

The credentials and other configs are loaded from AWS Secrets Manager secret "grafana"
Where config is stored like JSON, like this
```
{
  "api_token": "xxx",
  "url": "https://grafana.test.common-services.vydev.io/",
  "org_id": "1"
}
```

```
# Grafana API Token stored in Secrets Manager.
data "aws_secretsmanager_secret_version" "grafana" {
  secret_id = "grafana"
}

# needed by ecs-microservice module to create a Grafana Dashboard for microservice.
provider "grafana" {
  url    = jsondecode(data.aws_secretsmanager_secret_version.grafana.secret_string)["url"]
  auth   = jsondecode(data.aws_secretsmanager_secret_version.grafana.secret_string)["api_token"]
  org_id = jsondecode(data.aws_secretsmanager_secret_version.grafana.secret_string)["org_id"]
}
```

### CODE REVIEW

**Comment by dniel** [comment]
note that the dashboard template provided is not finished, and should be used as a starting point to create a much more advanced template.

**Comment by dniel** [comment]
It looks like it is enough to have "Editor" permission on the api-token. Should be limited, but existing tokens is not possible to change. Need to delete and recreate them and update secrets manager secret in each aws account with new token.

File: ecs-microservice/grafana.tf [hunk 1]
```diff
@@ -0,0 +1,29 @@
+###########################################################
+# Configure Grafana dashboard for microservice.
+#
+# Create a dashboard using the different values provided
+# as parameters to the mciroservice module to give a single
+# glass pane to watch the different components used by the
+# service.
+###########################################################
+
+data "aws_region" "current" {}
```
Comment (EirikRogno) [inline ecs-microservice/grafana.tf:10]: Hva trengs denne til?

Comment (dniel) [reply ecs-microservice/grafana.tf:10]: den var ment til å sende inn current region til templaten, men er ikke i bruk for øyeblikket.

**Review by stekern — APPROVED** [review]
Nice, looks good :+1:

**Pull request merged**